api/src/game_state_server.py

```diff
@@ -103,7 +107,9 @@ def new_connection_request(self, client_id: Hashable, room_id: str) -> Message:
                     except (WrongTypeError, MissingValueError, TypeError) as e:
                         # Don't raise here. Loading a room minus any tokens that have
                         # been corrupted is still valuable.
-                        print(e)
+                        logger.exception(
```

Comment:
Here you use logger.exception, but on line 176 you use logger.warning. and it feels like it should be the same level.

Reply:
I think this is appropriate. On line 176 we got a bad message from a client, there's nothing wrong with the server. Here there's something wrong with the server because rooms are corrupted

Reply:
On second thought though I'm going to change line 176 to logger.info because it's not really a warning if a client is sending invalid data, since anyone can do that any time